### PR TITLE
Add more manual tests

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tests/test_tools_manually.yml [hunk 1]
```diff
@@ -123,3 +123,24 @@
       md5sum: c09dff3f145d77d4848992e244811c08
     - path: results/variant_calling/mutect2/sample4_vs_sample3/sample4_vs_sample3.mutect2.vcf.gz.tbi
     # binary changes md5sums on reruns
+- name: Run full pipeline on tumoronly with most tools
+  command: nextflow run . -profile test --input tests/csv/3.0/fastq_tumor_only.csv --tools cnvkit,freebayes,merge,mpileup,mutect2,snpeff,strelka,tiddit,vep --outdir results
```
Comment (FriederikeHanssen) [inline tests/test_tools_manually.yml:127]: does snpeff and vep work here? because of 👨 vs 🪱 I mean?

Comment (maxulysse) [reply tests/test_tools_manually.yml:127]: yes

Comment (maxulysse) [reply tests/test_tools_manually.yml:127]: this is profile test, so the same data we use for the annotation test.
But to confirm, I tried the command again, and this works

Comment (FriederikeHanssen) [reply tests/test_tools_manually.yml:127]: hmmm I could have sworn we ran into issues before, that multiqc didn't run when combining step variantcalling with snpeff. anyways, good that it works!

Comment (FriederikeHanssen) [reply tests/test_tools_manually.yml:127]: 🤦‍♀️ Susanne fixed this mqc iirc and with the new release it should work now

**Review by FriederikeHanssen — APPROVED** [review]

File: tests/test_tools_manually.yml (hunk 1 above)
Comment (maxulysse) [reply tests/test_tools_manually.yml:127]: Oh yeah, I remember now

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please

**Pull request merged**